source/configure/enabling-chinese-japanese-korean-search.rst

```diff
@@ -3,9 +3,9 @@
 Chinese, Japanese, and Korean Search
 ======================================
 
-Enabling search for Chinese, Japanese and Korean (CJK) requires special configuration, since these languages do not contain spaces.
+Enabling search for Chinese, Japanese and Korean (CJK) requires a special configuration, since these languages do not contain spaces.
```

Comment:
```suggestion
Enabling search for Chinese, Japanese and Korean (CJK) requires special configuration, since these languages do not contain spaces.
```

Reply:
I removed the article "a" in this sentence because more than a single configuration setting is required to enable search for these key languages.